Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,11 @@
-import React from 'react';
-import './Header.css';
+import React from "react";
+import "./Header.css";
+
+const HeaderTitle = () => (
+  <div class="header-title">
+    <h3>Lambda School</h3>
```

Comment:
After passing props like I showed in my HeaderContent example, you would use it on line 6 like `<h3>{prop.h3Property}</h3>`.